blocks/algorithms/__init__.py

```diff
@@ -233,6 +236,8 @@ def __init__(self, step_rule=None, gradients=None, known_grads=None,
         self.total_step_norm = named_copy(l2_norm(self.steps.values()),
                                           "total_step_norm")
         self.on_unused_sources = on_unused_sources
+        self.theano_func_kwargs = theano_func_kwargs \
```

Comment:
I think using backslash rather than parens violates our style guide but I'm inclined to let it slide.